### DISCUSSION TITLE
Module pythainlp.translate

### DISCUSSION THREAD

**Discussion by ysirichom** [PyThaiNLP/pythainlp#574] [General]
Need help for Python module pythainlp.translate นั้นต้องใช้ memory เครื่องเท่าไหร่ค่ะ เพราะตอนทดสอบเรียก มี error resource_tracker.py:216: UserWarning: resource_tracker: There appear to be 1 leaked semaphore objects to clean up at shutdown ทุกครั้งรันไม่ได้เลย ดู memory ในเครื่องใช้ไปเกือบ 3 GB  ใครพอจะมีคำแนะนำไหมค่ะว่าต้องทำยังไงให้ทดสอบรัน pythainlp.translate ได้

**Comment by wannaphong** [comment]
ปัจจุบันนี้เราได้เพิ่มโมเดลใหม่ ๆ เข้าไปแล้วกับมี LLM ที่สามารถทำแปลภาษาได้ดีกว่าเดิม ดังนั้น ขอปิดกระทู้นี้นะครับ
